src/components/CareTeam.js

```diff
@@ -1,26 +1,111 @@
-
+import * as React from 'react';
 import DataGridComponent, { renderRequiredHeader } from './DataGridComponent';
+import Select from '@mui/material/Select';
+import MenuItem from '@mui/material/MenuItem';
+import FormControl from '@mui/material/FormControl';
+
 
-export const columns = providerOptions => [
+//keep columns for GFERequestPanel
+export const columns = props => [
```

Comment:
columns is used in GFERequestPanel only for the "Required" fields check. If the value for "required" is the same as that in "ourColumns", then delete this one as if anything changes, you need to modify the value in both places. The same comments for other xxxItem.

Reply:
Ok removing the additional 'columns' removes some of the error checking set up already -- specifically this can happen.
<img width="358" alt="Screen Shot 2022-08-05 at 3 29 35 PM" src="https://user-images.githubusercontent.com/106775284/183147826-93cf1728-5326-4ec8-b213-ca6f83ee072a.png">

Reply:
I was leaving 'columns' in until I found a way to export 'ourColumns' but not sure you can how it is set up. I will try to take out 'columns', and put in different error checking if that would be better.